server/lyft/decorators/destroy_plan_project_finder_wrapper.go

```diff
@@ -0,0 +1,47 @@
+package decorators
+
+import (
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/yaml/valid"
+	"github.com/runatlantis/atlantis/server/logging"
+	"path/filepath"
+	"strings"
+)
+
+type DestroyPlanProjectFinderWrapper struct {
+	events.ProjectFinder
+}
+
+func (d DestroyPlanProjectFinderWrapper) DetermineProjectsViaConfig(log logging.SimpleLogging, modifiedFiles []string, config valid.RepoCfg, absRepoDir string) ([]valid.Project, error) {
```

Comment:
Why do we need this wrapper? I think default ProjectFinder + StepRunner decorator should be enough to do what we need.

Reply:
My mistake, I flipped the pattern matching logic in my head when reading the code, we don't need this :)